### PR TITLE
Removes catgirls

### PR DESCRIPTION
**Pull request by optimumtact**
is now a braindamage line for gibbering

### CODE REVIEW

**Comment by Bawhoppen** [comment]
you got me

File: strings/brain_damage_lines.json [hunk 1]
```diff
@@ -57,7 +57,8 @@
         "i will snatch erry motherfucker birthday",
         "the ai and borgs are mettacomming I think",
         "u just did the world a little bit more sad place for someone",
-        "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?"
+        "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?",
+        "Remove KAT girlsss"
```
Comment (RemieRichards) [inline strings/brain_damage_lines.json]: Remve pls.

**Comment by Anonmare** [comment]
Have a tail? Go to jail!
Splat the cat!
Shoot with a beeline for the feline.
Have no fears, remove their ears
Anime is banime

**Comment by CPTANT** [comment]
Didn't we have a discussion about misleading PR titles?

**Comment by Bawhoppen** [comment]
@CPTANT the rules don't apply to oranges, silly

**Comment by Thunder12345** [comment]
Misleading titles in PRs adding brain damage lines is a long held tradition

**Comment by lzimann** [comment]
The brain damage line should be "I like catgirls" instead of this.

**Comment by RemieRichards** [comment]
>leo upset that he's braindamaged

😆

**Comment by lzimann** [comment]
Hey I am not the one that likes cat girls or use emojis!

**Comment by RemieRichards** [comment]
😸

**Comment by calzilla1** [comment]
Remove quat grills

**Comment by CosmicScientist** [comment]
Do sneks say girlsss or girlsssssssss with this?

**Comment by PKPenguin321** [comment]
Give tattoos

**Comment by ShadowDeath6** [comment]
Got my hopes up :^(

**Pull request closed without merging**